server/gdial-rest.c

```diff
@@ -571,7 +571,7 @@ static void gdial_rest_server_handle_GET_app(GDialRestServer *gdial_rest_server,
   if(allow_stop == NULL) {
     allow_stop = "false";
   }
-  g_print("server_register_application allowStop:%s\n",allow_stop);
+  g_print("server_register_application allowStop:%s\n",(gchar*)allow_stop);
```

Comment:
If allow_stop is gchar* then it should be declared as gchar*. Is it even used somewhere?

Reply:
good point; it doesn't seem to be used (other than for this log) - but probably makes more sense to make it gchar*